src/redux/sagas/colony/colonyCreate.ts

```diff
@@ -529,6 +545,31 @@ function* colonyCreate({
        */
       yield put(transactionPending(setOneTxRoleAdministration.id));
 
+      /*
+       * Generate proofs for setting permissions the the newly deployed OneTxPayment extension
+       */
+      const colonyClient = yield colonyManager.getClient(
+        ClientType.ColonyClient,
+        colonyAddress,
+      );
+
+      let oneTxPaymentRoleProofs;
+      // This has two potential permissions, so we try both of them
+      try {
+        oneTxPaymentRoleProofs = yield getPermissionProofs(
+          colonyClient,
+          Id.RootDomain,
+          ColonyRole.Architecture,
```

Comment:
Can you explain this? Under what circumstances would one fail and not the other?

Reply:
Yeah, if I think real hard about it, this should just be root in this context.

Don't get me wrong, the logic is still correct, as you can assign permissions to this extension both with Architecture or Root. 

However in the context of this saga, this will be a newly deployed colony, and this a newly deployed extension, so the current user's address will always have Root, since they were the ones that just deployed the colony.